### PR TITLE
Lots of removal! Mostly Keypool and few commands related to the stackref and queueref

### PR DESCRIPTION
**Pull request by AshwinKul28**
Removed keypool from the store and refactored code accordingly.
Removed Following commands and their implementations: 
1. stackref
2. stackint
3. queueref
4. queueint

### CODE REVIEW

**Comment by AshwinKul28** [comment]
Benchmarks
[dice_benchmark_keypool_remove.txt](https://github.com/user-attachments/files/16894840/dice-benchmark_refactored.txt)

File: core/object.go [hunk 1]
```diff
@@ -19,10 +19,8 @@ var ObjEncodingEmbStr uint8 = 8
 
 var ObjTypeByteList uint8 = 1 << 4
 var ObjEncodingQint uint8 = 0
```
Comment (lucifercr07) [inline core/object.go:21]: We can remove this too.

File: core/object.go [hunk 2]
```diff
@@ -19,10 +19,8 @@ var ObjEncodingEmbStr uint8 = 8
...
-var ObjEncodingQref uint8 = 1
 
 var ObjEncodingStackInt uint8 = 2
```
Comment (lucifercr07) [inline core/object.go:23]: We can remove this too.

**Review by lucifercr07 — APPROVED** [review]
LGTM, minor comments

File: core/object.go (hunk 2 above)
Comment (AshwinKul28) [reply core/object.go:23]: Thanks, sir. Done.

File: core/object.go (hunk 1 above)
Comment (soumya-codes) [inline core/object.go:21]: This last one left out...

File: core/object.go (hunk 2 above)
Comment (soumya-codes) [inline core/object.go:23]: This as well..

**Review by soumya-codes — COMMENTED** [review]
Left couple of minor comments similar to that of @lucifercr07 .
Otherwise looks good.

Thanks for the mammoth effort.
This would help my refactoring effort a great deal.
Many thanks.

**Review by soumya-codes — APPROVED** [review]
LGTM

**Review by JyotinderSingh — APPROVED** [review]

**Pull request merged**